### ISSUE TITLE
Site not responsive

### ISSUE THREAD

**Issue by preritk** [preritk/Personal-Website#3]
Site is not fitting in view of different devices.

**Comment by preritk** [comment]
We need to resolve the issue so that the website it becomes responsive to the screen size of the host display device.
![WhatsApp Image 2020-08-24 at 11 53 45 PM (2)](https://user-images.githubusercontent.com/30365876/91124375-bef21e00-e6bc-11ea-8b85-864e0afdf174.jpeg)------>The Normal View as seen on a regular desktop.

![WhatsApp Image 2020-08-24 at 11 53 45 PM (1)](https://user-images.githubusercontent.com/30365876/91124508-feb90580-e6bc-11ea-964e-6dc531f3f95a.jpeg) -------------->The view in a Medium Sized Android Screen
![WhatsApp Image 2020-08-24 at 11 53 45 PM](https://user-images.githubusercontent.com/30365876/91124540-0c6e8b00-e6bd-11ea-9a58-42d2fda62d04.jpeg) ------------->The view in a Large Sized Android Screen

**Comment by rishabhgarg15** [comment]
@prertik I would like to work on this issue.

**Comment by preritk** [comment]
@rishabhgarg15  if you understood the issue, go ahead and submit a PR.

**Comment by rishabhgarg15** [comment]
if I am understanding the issue correctly the acceptance test of this bug is that the website should become responsive for all extensively available test screens.

**Comment by preritk** [comment]
@rishabhgarg15, yes that it would be the acceptance criteria for your PR. Go ahead

**Comment by Garvitmodi** [comment]
Hi @preritk ,I and @rishabhgarg15 were working on this issue. And we had made a Code Style Guide , which we will be following. It's general Code Style Guide. I am sharing it with you, please review and reply can we go with it.
[code_style_guide.docx](https://github.com/preritk/Personal-Website/files/5512259/code_style_guide.docx)

**Comment by Garvitmodi** [comment]
Hi @preritk , I think that vertical line in contact page, should have display  none when width became less than 854px. 
So, I if you like that please give as approval for doing this. As their is no need of vertical line when both grid items came in separate rows. I am adding Screen Shots for width> 855px, for width < 855 px ,and for width = 855px. Please reply what's your suggestion for this.
![Screenshot (66)](https://user-images.githubusercontent.com/44116230/98579246-5b06eb00-22e4-11eb-981a-77ce6d30bf17.png)
![Screenshot (67)](https://user-images.githubusercontent.com/44116230/98579256-5fcb9f00-22e4-11eb-99d7-abc3a8a0f1a5.png)
![Screenshot (69)](https://user-images.githubusercontent.com/44116230/98579258-61956280-22e4-11eb-8b09-26a3286e59ad.png)

**Comment by preritk** [comment]
@Garvitmodi and @rishabhgarg15 , I had read the Code Style Guide , it is fine by me. You can go ahead with it.

**Comment by preritk** [comment]
@Garvitmodi and @rishabhgarg15 , the changes suggested on contact page are fine by my side. You can proceed with them.

**Comment by Garvitmodi** [comment]
@preritk , Thanks for your review.

**Comment by preritk** [comment]
Thanks @rishabhgarg15 and @Garvitmodi for the quick fix!!
